flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -645,6 +645,10 @@
 	@Deprecated
 	public static final String JOB_MANAGER_WEB_LOG_PATH_KEY = "jobmanager.web.log.path";
 
+	/** Config parameter indicating whether jobs can be uploaded and run from the web-frontend. */
+	public static final ConfigOption<Boolean> JOB_MANAGER_WEB_ACCESSLOG_ENABLE =
+			key("jobmanager.web.accesslog.enable").defaultValue(false);
```

Comment:
please move the `.defaultValue` call into a new line. It should also be moved to the JobManagerOptions class.